### PR TITLE
Fix a shape inference bug for FusedConv and MaxpoolWithMask

### PR DESCRIPTION
**Pull request by duli2012**
Make fusedconv shape inference to be the same as Conv.
Make MaxpoolWithMask shaper inference to be the same as Maxpool.

### CODE REVIEW

**Comment by askhade** [comment]
Looks good to me
FYI - In order to enable ConvActivationFusion you need more changes... Not sure if you are planning to add it as part of another PR

**Comment by duli2012** [comment]
> Looks good to me
> FYI - In order to enable ConvActivationFusion you need more changes... Not sure if you are planning to add it as part of another PR

Just checked in code to enable this tranformer.

File: onnxruntime/core/session/inference_session.h [hunk 1]
```diff
@@ -67,7 +67,7 @@ struct SessionOptions {
   unsigned max_num_graph_transformation_steps = 5;  // TODO choose a good default here?
 
   // set graph optimization level
-  TransformerLevel graph_optimization_level = TransformerLevel::Level1;
+  TransformerLevel graph_optimization_level = TransformerLevel::Level2;
```
Comment (askhade) [inline onnxruntime/core/session/inference_session.h]: Don't change the default level...

Comment (duli2012) [reply onnxruntime/core/session/inference_session.h]: Done.

**Pull request merged**